packages/code-display/src/CodeDisplay.tsx

```diff
@@ -112,4 +123,22 @@ export const CodeDisplay: FC<PrismEditorProps> = ({
       )}
     </Highlight>
   )
+  if (!onChange) {
+    return highlight(code)
```

Comment:
is this for when `CodeDisplay` is used as an editor? If so I think a comment would be handy